diff --git a/node_modules/qiankun/es/sandbox/patchers/index.js b/node_modules/qiankun/es/sandbox/patchers/index.js
index af2721d..cda425b 100644
--- a/node_modules/qiankun/es/sandbox/patchers/index.js
+++ b/node_modules/qiankun/es/sandbox/patchers/index.js
@@ -10,6 +10,7 @@ import patchHistoryListener from './historyListener';
 import patchInterval from './interval';
 import patchWindowListener from './windowListener';
 import patchUIEvent from './UIEvent';
+console.log('嘿嘿');
 export function patchAtMounting(appName, elementGetter, sandbox, singular) {
   var _patchersInSandbox;
 
